docs/examples/table/main.js

```diff
@@ -9,11 +9,6 @@ export default function Example(): Node {
         <Table accessibilityLabel="Main example table">
           <Table.Header>
             <Table.Row>
-              <Table.HeaderCell>
```

Comment:
This example is pretty broken rn 
![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/234421214-91a136e2-43d4-4c11-92f5-f4e8a0d4b1b8.png)